src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -292,10 +424,18 @@ public Server build() {
             blockingTaskExecutor = defaultBlockingTaskExecutor();
         }
 
+        final List<ServerPort> ports =
```

Comment:
Add to port()'s docstring that if not called, a default of 0 (randomly-assigned port) and SessionProtocol.HTTP will be used.